### ISSUE TITLE
Allow skipping of onboarding on first launch of the app.

### ISSUE THREAD

**Issue by I2ujbNGakP8KfLKbho0Z** [tachiyomiorg/tachiyomi#10270]
### Describe your suggested feature

Certain (mostly specialized) older Android devices do not ship with a default file picker (com.android.documentsui). This means that it is impossible to complete the onboarding folder selection step for these devices.

When clicking on "Select a folder", an error message saying "No file picker app found" appears and nothing else happens.

This could be resolved by either allowing manual entry of a file path for Tachiyomi's default folder, or by allowing this step to be skipped altogether. This option could perhaps be hidden under an "Advanced Options" button that explicitly warns the user of the problems that may occur if they choose to skip this step.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have updated the app to version **[0.14.7](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by arkon** [comment]
I don't think this will realistically ever make sense to do, since mandating a storage location be set ends up making everything else much simpler (and it's not quite as simple as just writing some file path, the system handles permissions).

If you're on an old enough version of Android, you might have some luck with just installing third-party file picker apps and setting it as your default (Root Explorer, Solid Explorer, Files by Google, etc.). If you're on a relatively newer version of Android that didn't ship with a file explorer, then that's just an intentionally cursed system I don't particularly have interest in supporting. If somebody finds this issue and you realized you removed your system file picker app while "cleaning it", then you should probably undo that.

I could _sort_ of see someone wanting to just use the app without any storage functionality (i.e. never downloading, using local source, or having backups), but that seems like even less than the ~1.5% of users using Android < 8 that we're dropping soon...

**Comment by liu-jianyang** [comment]
Just saw this issue show up as well for me (Onyx Boox). I think (according to https://www.reddit.com/r/NeatBytes/comments/zkbpm4/any_way_to_make_solid_explorer_as_default_file/) there's no way to pick a default file manager, so if Android doesn't ship with a default file manager, then I'm f*cked.
`Whenever an app calls for the user to pick a file, the default file manager (Files) opens and from there you should be able to call up Solid Explorer. `
Given that you're not dropping support for Android < 8 yet, would it be too much of an ask to give us a version that does not mandate the storage location? The rest of the features (in particular eink refresh and rotate to landscape for spreads) are ones that I'd love to use that do not require Android 8+.
